lgc/elfLinker/GlueShader.cpp

```diff
@@ -34,6 +34,7 @@
 #include "FetchShader.h"
 #include "NullFragmentShader.h"
 #include "lgc/state/PassManagerCache.h"
+#include "llvm-dialects/Dialect/Dialect.h"
```

Comment:
I think you might need to edit .clang_format so it understands the ordering of this include file.